workers/unity/Packages/com.improbable.gdk.gameobjectcreation/SpatialOSEntity.cs

```diff
@@ -20,12 +21,12 @@ internal SpatialOSEntity(Entity entity, EntityManager entityManager)
             SpatialOSEntityId = entityManager.GetComponentData<SpatialEntityId>(entity).EntityId;
         }
 
-        public bool HasComponent<T>() where T : struct, ISpatialComponentData, IComponentData
```

Comment:
If we are removing that then we should also be giving access to other component types. Either we only guarantee that the Spatial components are there and we just give a user that, or we give them full access to the entity. This might come down to how much we want to be able to use this object elsewhere.

Reply:
The purpose of this is to be the thing you use when you don't want to touch the ECS though, so restricting it might just make things easier.

Reply:
discussed offline: reverting it for now as we don't have any use case where we would want to get a component that is not an `ISpatialComponentData`